app/assets/css/utilities.css

```
@@ -1,3 +1,36 @@
+/* Whitney — Discord brand font loaded from Skyra CDN */
+@font-face {
+	font-family: "Whitney";
```

Stylelint (17.10.0)

[error] Expected no quotes around "Whitney" (font-family-name-quotes)

(font-family-name-quotes)